cluster/variables.tf

```diff
@@ -145,3 +145,23 @@ variable "dns_provider" {
   description = "DNS provider to use for the cluster."
   default     = "KubeDNS"
 }
+
+variable "k8s_image_encryption" {
+  description = "Enable k8s image encryption"
+  default     = false
+}
+
+variable "etcd_encrypted_volumes" {
+  description = "Enable etcd volume encryption"
+  default     = "false"
+}
+
+variable "etcd_encryption_kms_key" {
```

Comment:
as with the storage class, you need the full key ARN for kops, so I would make it explicit in the name (`etcd_encryption_kms_key`) and in the description.

Btw, why not use the same key for both etcd and the root volume?

Reply:
I think it gives more configuration freedom.